### ISSUE TITLE
Unable to import sciunit utils 0.2.7 on Python 3.11

### ISSUE THREAD

**Issue by mdenker** [scidash/sciunit#219]
Importing sciunit 0.2.7 on Python 3.11 fails with the following output

```python
>>> from sciunit import utils
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/__init__.py", line 16, in <module>
    from .scores import Score
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/scores/__init__.py", line 7, in <module>
    from .complete import *
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/scores/complete.py", line 12, in <module>
    from sciunit import errors, utils
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/utils.py", line 814, in <module>
    generic_arg_hash_function = fn_arg_hash_function(_generic_fn)
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/utils.py", line 755, in fn_arg_hash_function
    default_kwargs = fn_kwargs(fn)
                     ^^^^^^^^^^^^^
  File "miniconda3/envs/nwtest/lib/python3.11/site-packages/sciunit/utils.py", line 733, in fn_kwargs
    (args, _, _, defaults) = inspect.getargspec(fn)
                             ^^^^^^^^^^^^^^^^^^
AttributeError: module 'inspect' has no attribute 'getargspec'. Did you mean: 'getargs'?
```

The same statement works fine for me on earlier Python versions. The error seems to be related to this potentially easily fixable deprecation https://stackoverflow.com/questions/76151813/error-module-inspect-has-no-attribute-getargspec

**Comment by rgerkin** [comment]
Can you try and see if replacing `inspect.getargspec(fn)` with `inspect.getfullargspec(fn)` fixes it?

**Comment by mdenker** [comment]
Ah, ok, sorry, I didn't realize the current master already has this included from #200 -- this fixes importing issues. Thanks!
